core/src/test/java/org/elasticsearch/index/query/SimpleQueryStringBuilderTest.java

```diff
@@ -177,15 +178,6 @@ public void testNullQueryTextGeneratesException() {
         assertThat(exception, notNullValue());
     }
 
-    @Test
-    public void testHandlingDefaults() throws IOException {
```

Comment:
to be honest I don't get what this test was doing... hence I didn't know what to replace it with given that createExpectedQuery is gone...